Toggl.Core.UI/ViewModels/Reports/ReportsViewModel.cs

```diff
@@ -26,6 +26,7 @@ namespace Toggl.Core.UI.ViewModels.Reports
 {
     public sealed class ReportsViewModel : ViewModel
     {
+        private static readonly string dropdownCharacter = "▾";
```

Comment:
```suggestion
        private const string dropdownCharacter = "▾";
```

use `static readonly` only when you have to actually calculate (or get) the value from the constructor.